arches_lingo/src/arches_lingo/api.ts

```diff
@@ -59,6 +59,27 @@ export const fetchSchemeCreation = async (schemeId: string) => {
     return parsed;
 };
 
+export const fetchSchemeLabel = async (schemeId: string) => {
+    const response = await fetch(arches.urls.api_scheme_label(schemeId));
+    const parsed = await response.json();
+    if (!response.ok) throw new Error(parsed.message || response.statusText);
+    return parsed;
+};
+
+export const deleteSchemeLabelTile = async (tileId: string) => {
+    const response = await fetch(arches.urls.api_scheme_label_tile(tileId), {
+        method: "DELETE",
+        headers: { "X-CSRFTOKEN": getToken() },
+    });
+
+    if (!response.ok) {
+        const parsed = await response.json();
+        throw new Error(parsed.message || response.statusText);
+    } else {
+        return true;
```

Comment:
If the next logical question here is why are deletes returning 204 No Content instead of 200 OK, I guess I don't think we should adjust the semantics of the HTTP response just because our current patterns for frontend fetching will look more similar if we always return 200. What if we in the future adopt a global API client that assumes in the absence of further configuration that deletes return 204s? We will have changed the semantics of the API for no reason.

Plus, we have these separate fetch methods anyway, may as well use them (otherwise it really does start to make me wonder if we should use a global API client).

Reply:
no yeah, you have a good point. Resolving 👍